app/views/works/_standard_form.html.erb

```diff
@@ -1,40 +1,43 @@
 <!--REVIEW: trying out articles on these fieldsets to see if that's a better chunker, but may revert after testing. Have also tossed in landmark headings on top level groups-->
```

Comment:
Partially. I'm conflicted about the article roles and what to do about them, but the landmark headings are standard operating procedure for us at this point.